src/metabase/api/table.clj

```diff
@@ -293,11 +298,15 @@
   Passing `include_hidden_fields=true` will include any hidden `Fields` in the response. Defaults to `false`
   Passing `include_sensitive_fields=true` will include any sensitive `Fields` in the response. Defaults to `false`.
 
+  Passing `include_editable_data_model=true` will check that the current user has write permissions for the table's
+  data model, rather than checking that they have data access perms for the table.
+
   These options are provided for use in the Admin Edit Metadata page."
-  [id include_sensitive_fields include_hidden_fields]
+  [id include_sensitive_fields include_hidden_fields include_editable_data_model]
```

Comment:
the arg list seems to grow, I'd change it to an option maps for all the include_* arg

Reply:
oh sorry I referred to the wrong fn, I meant the arg list for `fetch-query-metadata`